docker/webui/haproxy-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I recommend for shell scripts to always use `-e`

Reply:
Good idea